globalvars.py

```diff
@@ -12,11 +12,13 @@
 # noinspection PyCompatibility
 import regex
 import subprocess as sp
+import json
```

Comment:
It's unclear to me why the json package is being imported. There don't appear to be any lines added or changed which use the json package.

Reply:
Leftovers from trying to parse strings then I saw the .getboolean and .getfloat options in configparser.  will remove